web/docs/tql2/functions/keys.md

```diff
@@ -0,0 +1,34 @@
+# keys
+
+Retrieves a list of field names from a record.
+
+```tql
+keys(x:record) -> list<string>
+```
+
+## Description
+
+The `keys` function returns a list of strings containing all field names from the input record `x`.
+
+### `x: record`
+
+The record whose field names you want to retrieve.
+
+## Examples
+
+### Get all field names from a record
```

Comment:
Can you all this bend-over-backwards example of `this[this.keys().sort().first()]`?

Reply:
I'm not sure that's a good idea. The easier way to write that is just `this.sort()[0]?`, which I think is a lot easier to read.

Reply:
Ah well, I did end up adding it and also the example from Discord.

Reply:
Love the new example! This could even be plotted as a bar chart now.